code/modules/projectiles/projectile.dm

```diff
@@ -172,6 +172,8 @@
 			if(isalien(L))
 				new /obj/effect/temp_visual/dir_setting/bloodsplatter/xenosplatter(target_loca, splatter_dir)
 			var/obj/item/bodypart/B = L.get_bodypart(def_zone)
+			if(!B.status)
+				return
```

Comment:
jesteś pewien z tym? nie powinieneś sprawdzać `B`?

Reply:
Racja, zaraz to poprawie